packages/foundry-sdk-generator/src/generate/betaClient/copyFiles.ts

```diff
@@ -122,37 +83,81 @@ export async function copyFiles(
         fileContents,
         { overwrite: true },
       );
-      for (const importDeclaration of sourceFile.getImportDeclarations()) {
```

Comment:
All of this code was identical to the extract method, except the extracted method also had the changes for `importSet`. Therefore, that method is adjusted to not add to `importSet` if it is undefined to make these consistent.